proto/spire/next/api/entry/v1/entry.proto

```diff
@@ -6,28 +6,29 @@ import "google/protobuf/empty.proto";
 import "spire/next/types/entry.proto";
 import "spire/next/types/selector.proto";
 
+// Manages registration entries stored by the SPIRE Server.
 service Entry {
     // Lists entries.
     rpc ListEntries(ListEntriesRequest) returns (ListEntriesResponse);
 
-    // Gets an entry by id.
+    // Gets an entry by ID. If the entry does not exist, NOT_FOUND is returned.
     rpc GetEntry(GetEntryRequest) returns (spire.types.Entry);
 
-    // Creates an entry.
+    // Creates an entry. If an entry like it already exists, ALREADY_EXISTS is returned.
     rpc CreateEntry(CreateEntryRequest) returns (spire.types.Entry);
 
     // Creates one or more entries in a batch.
     rpc BatchCreateEntry(BatchCreateEntryRequest) returns (BatchCreateEntryResponse);
 
-    // Updates an entry.
+    // Updates an entry. If the entry ID does not exist, NOT_FOUND is returned.
     rpc UpdateEntry(UpdateEntryRequest) returns (spire.types.Entry);
 
-    // Deletes the entry identified by the request. If the entry does not exist
-    // the call returns NOT_FOUND.
+    // Deletes the entry identified by the request. If the entry does not exist,
+    // NOT_FOUND is returned.
     rpc DeleteEntry(DeleteEntryRequest) returns (google.protobuf.Empty);
 
-    // Deletes the entries identified by the request. 
-    rpc BatchDeleteEntry(BatchDeleteEntryRequest) returns (google.protobuf.Empty);
+    // Deletes one or more entries identified by the request.
+    rpc BatchDeleteEntry(BatchDeleteEntryRequest) returns (BatchDeleteEntryResponse);
```

Comment:
We need a new message for BatchDeleteResponse because we need to indicate which deletes failed (if any).

Reply:
Yes. Good.